client/core/trade.go

```diff
@@ -975,7 +1025,7 @@ func (t *trackedTrade) revokeMatch(matchID order.MatchID) error {
 	// Notify the user of the failed match.
 	corder, _ := t.coreOrderInternal() // no cancel order
 	t.notify(newOrderNote("Match revoked", fmt.Sprintf("Match %s has been revoked",
-		t.token()), db.WarningLevel, corder))
+		matchID), db.WarningLevel, corder))
```

Comment:
Turns out 64-character strings look pretty bad in a 450 px wide notification tray. Until we can get more sophisticated with this, can we use `token(matchID[:])`.